src/test/cli.build.test.ts

```diff
@@ -27,6 +27,20 @@ describe('Dev Containers CLI', function () {
 
 	describe('Command build', () => {
 
+		it('should fail to push --image-name with --push true', async () => {
+			let success = false;
+			const testFolder = `${__dirname}/configs/example-push`;
+			try {
+				await shellExec(`${cli} build --workspace-folder ${testFolder} --image-name demo:v1 --push true`);
+			} catch (error) {
+				assert.equal(error.error.code, 1, 'Should fail with exit code 1');
+				const res = JSON.parse(error.stdout);
+				assert.equal(res.outcome, 'error');
+				assert.match(JSON.stringify(error.error.message), /unauthorized: access token has insufficient scopes/);
```

Comment:
Where are we testing if the image which the CLI tries to push is `demo:v1` ?

Reply:
added one more match statement to check it is pushing correct image name which we are passing through --image-name